gulpfile.js

```diff
@@ -4,51 +4,51 @@
 
 /* global __dirname, require, process */
 
-var gulp = require('gulp');
-var karma = require('karma');
-var eslint = require('gulp-eslint');
-var watch = require('gulp-watch');
-var spawn = require('child_process').spawn;
+const gulp = require('gulp');
+const karma = require('karma');
+const eslint = require('gulp-eslint');
+const watch = require('gulp-watch');
+const spawn = require('child_process').spawn;
 
-var lintPaths = [
+const lintPaths = [
     'media/js/**/*.js',
     '!media/js/libs/*.js',
     'tests/unit/spec/**/*.js',
     'gulpfile.js'
 ];
 
-gulp.task('serve:backend', function () {
-    var devServerPort = process.env.PORT || 8000;
+gulp.task('serve:backend', () => {
+    const devServerPort = process.env.PORT || 8000;
     process.env.PYTHONUNBUFFERED = 1;
     process.env.PYTHONDONTWRITEBITECODE = 1;
-    spawn('python', ['manage.py', 'runserver', '0.0.0.0:' + devServerPort], {
+    spawn('python', ['manage.py', 'runserver', `0.0.0.0:${devServerPort}`], {
         stdio: 'inherit'
     });
 });
 
-gulp.task('media:watch', function () {
+gulp.task('media:watch', () => {
     return gulp.src('./media/**/*')
         .pipe(watch('./media/**/*', {
             'verbose': true
         }))
         .pipe(gulp.dest('./static'));
 });
 
-gulp.task('js:test', function(done) {
+gulp.task('js:test', (done) => {
```

Comment:
I'm a bit torn on this one too. On one hand it saves a couple of bytes, but on the other hand it feels a bit inconsistent as parens are required at all other times, including when there are no arguments. I'm going to omit them for now as you pointed it out, but I've still not quite decided what my preference is here either.